roles/ldap-perun/tasks/install-Debian.yml

```diff
@@ -41,6 +41,7 @@
     olcSuffix: "{{ slapd_basedn }}"
     olcRootDN: "{{ slapd_basedn_admin }}"
     olcRootPW: "{{ root_password.stdout }}"
+  notify: Restart LDAP
```

Comment:
indentation

Reply:
This notification is called after completion of ldap_attr command and I think it should has same indentation as name of this Ansible module.

Reply:
Make sense, then it looks OK. Since I believe you tested it on new instance, we can merge.